src/controllers/dynakube/dynakube_controller.go

```diff
@@ -129,7 +129,7 @@ func (controller *Controller) setRequeueAfterIfNewIsShorter(requeueAfter time.Du
 
 func isDynatraceAPIUnreachable(err error) bool {
 	var serverErr dtclient.ServerError
-	if errors.As(err, &serverErr) && (serverErr.Code == http.StatusTooManyRequests || serverErr.Code == http.StatusServiceUnavailable) {
+	if dynatraceapi.IsUnreachable(err) {
```

Comment:
any reason why we need to warp the online function into another one? Should we move the log to case and make only 1 function instead of two?

Reply:
i mean:
```
switch {
	case dynatraceapi.IsUnreachable(err) (err):
		log.Info("dynaTrace API server is unavailable or request limit reached! trying again in one minute",
			"errorCode", serverErr.Code, "errorMessage", serverErr.Message)
		return reconcile.Result{RequeueAfter: fastUpdateInterval}, nil
```

Reply:
Just to keep the (condition) logic in one place.